scripts/analyze

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import json
+import re
+import shutil
+
+force = ['']
+featurefile = ''
+
+for i in sys.argv:
+    if i == sys.argv[0]:
+        continue
+    option = i.split('=')[0]
+    value = i.split('=')[1]
```

Comment:
Even better: 
~~~
option, value = i.split('=', 1)
~~~
will allow to have values with "=" inside (2nd arg to split() is how many splits to make, not how many parts to get).